lib/galaxy/jobs/__init__.py

```diff
@@ -1743,7 +1743,14 @@ def _change_ownership( self, username, gid ):
         job = self.get_job()
         # FIXME: hardcoded path
         external_chown_script = self.get_destination_configuration("external_chown_script", None)
-        cmd = [ '/usr/bin/sudo', '-E', external_chown_script, self.working_directory, username, str( gid ) ]
+
+        cmd = [ '/usr/bin/sudo', '-E', ]
```

Comment:
I dont get what you are suggesting. We could have a little bash helper/wrapper: 

dosudo.sh:

#!/bin/bash
sudo -E AND_WHATEVER_IS_NEEDED "$@" 

From python we could then call: dosudo.sh externalhelper.py ...